api/bucketsd/client/client.go

```diff
@@ -370,11 +370,9 @@ func (c *Client) Archive(ctx context.Context, key string, opts ...ArchiveOption)
 	return err
 }
 
-// ArchiveStatus returns the status of a Filecoin bucket archive.
-func (c *Client) ArchiveStatus(ctx context.Context, key string) (*pb.ArchiveStatusResponse, error) {
-	return c.c.ArchiveStatus(ctx, &pb.ArchiveStatusRequest{
-		Key: key,
-	})
+// Archives returns information about current and historical archives.
+func (c *Client) Archives(ctx context.Context, key string) (*pb.ArchivesResponse, error) {
```

Comment:
Combining those two old methods into one here. `Archives` returns an object that looks like:
```
{
  Current: nil|Archive,
  History: Archive[]
}
```
If the bucket has never been archived, both those values are the nil/empty values.
If the bucket has an archive currently processing, it will be listed in `Current` and is updates in near-real-time as new deal information is available. The previously completed archive is moved to the first slot in `History`.

Reply:
I added this comment here in the client, but the change is reflected all the way through the stack.